File: php/SwaggerClient-php/lib/Model/VaultCheck.php
<?php

namespace Swagger\Client\Model;

use \ArrayAccess;
use \Swagger\Client\ObjectSerializer;

class VaultCheck implements ModelInterface, ArrayAccess
{
    const DISCRIMINATOR = null;

    protected static $swaggerModelName = 'VaultCheck';

    protected static $swaggerTypes = [
        'response' => '\Swagger\Client\Model\Response',
        'check_id' => 'int',
        'main' => 'bool',
        'check_routing' => 'string',
        'check_account' => 'string',
        'check_number' => 'string',
        'check_first_name' => 'string',
        'check_last_name' => 'string',
        'note' => 'string',
        'masked_routing' => 'string',
        'masked_account' => 'string',
        'account_type' => 'string'
    ];

    protected static $swaggerFormats = [
        'response' => null,
        'check_id' => 'int32',
        'main' => null,
        'check_routing' => null,
        'check_account' => null,
        'check_number' => null,
        'check_first_name' => null,
        'check_last_name' => null,
        'note' => null,
        'masked_routing' => null,
        'masked_account' => null,
        'account_type' => null
    ];

    public static function swaggerTypes()
    {
        return self::$swaggerTypes;
    }

    public static function swaggerFormats()
    {
        return self::$swaggerFormats;
    }

    protected static $attributeMap = [
        'response' => 'Response',
        'check_id' => 'CheckID',
        'main' => 'Main',
        'check_routing' => 'CheckRouting',
        'check_account' => 'CheckAccount',
        'check_number' => 'CheckNumber',
        'check_first_name' => 'CheckFirstName',
        'check_last_name' => 'CheckLastName',
        'note' => 'Note',
        'masked_routing' => 'MaskedRouting',
        'masked_account' => 'MaskedAccount',
        'account_type' => 'AccountType'
    ];

    protected static $setters = [
        'response' => 'setResponse',
        'check_id' => 'setCheckId',
        'main' => 'setMain',
        'check_routing' => 'setCheckRouting',
        'check_account' => 'setCheckAccount',
        'check_number' => 'setCheckNumber',
        'check_first_name' => 'setCheckFirstName',
        'check_last_name' => 'setCheckLastName',
        'note' => 'setNote',
        'masked_routing' => 'setMaskedRouting',
        'masked_account' => 'setMaskedAccount',
        'account_type' => 'setAccountType'
    ];

    protected static $getters = [
        'response' => 'getResponse',
        'check_id' => 'getCheckId',
        'main' => 'getMain',
        'check_routing' => 'getCheckRouting',
        'check_account' => 'getCheckAccount',
        'check_number' => 'getCheckNumber',
        'check_first_name' => 'getCheckFirstName',
        'check_last_name' => 'getCheckLastName',
        'note' => 'getNote',
        'masked_routing' => 'getMaskedRouting',
        'masked_account' => 'getMaskedAccount',
        'account_type' => 'getAccountType'
    ];

    public static function attributeMap()
    {
        return self::$attributeMap;
    }

    public static function setters()
    {
        return self::$setters;
    }

    public static function getters()
    {
        return self::$getters;
    }

    public function getModelName()
    {
        return self::$swaggerModelName;
    }

    protected $container = [];

    public function __construct(array $data = null)
    {
        $this->container['response'] = isset($data['response']) ? $data['response'] : null;
        $this->container['check_id'] = isset($data['check_id']) ? $data['check_id'] : null;
        $this->container['main'] = isset($data['main']) ? $data['main'] : null;
        $this->container['check_routing'] = isset($data['check_routing']) ? $data['check_routing'] : null;
        $this->container['check_account'] = isset($data['check_account']) ? $data['check_account'] : null;
        $this->container['check_number'] = isset($data['check_number']) ? $data['check_number'] : null;
        $this->container['check_first_name'] = isset($data['check_first_name']) ? $data['check_first_name'] : null;
        $this->container['check_last_name'] = isset($data['check_last_name']) ? $data['check_last_name'] : null;
        $this->container['note'] = isset($data['note']) ? $data['note'] : null;
        $this->container['masked_routing'] = isset($data['masked_routing']) ? $data['masked_routing'] : null;
        $this->container['masked_account'] = isset($data['masked_account']) ? $data['masked_account'] : null;
        $this->container['account_type'] = isset($data['account_type']) ? $data['account_type'] : null;
    }

    public function listInvalidProperties()
    {
        $invalidProperties = [];

        return $invalidProperties;
    }

    public function valid()
    {
        return count($this->listInvalidProperties()) === 0;
    }

    public function getResponse()
    {
        return $this->container['response'];
    }

    public function setResponse($response)
    {
        $this->container['response'] = $response;

        return $this;
    }

    public function getCheckId()
    {
        return $this->container['check_id'];
    }

    public function setCheckId($check_id)
    {
        $this->container['check_id'] = $check_id;

        return $this;
    }

    public function getMain()
    {
        return $this->container['main'];
    }

    public function setMain($main)
    {
        $this->container['main'] = $main;

        return $this;
    }

    public function getCheckRouting()
    {
        return $this->container['check_routing'];
    }

    public function setCheckRouting($check_routing)
    {
        $this->container['check_routing'] = $check_routing;

        return $this;
    }

    public function getCheckAccount()
    {
        return $this->container['check_account'];
    }

    public function setCheckAccount($check_account)
    {
        $this->container['check_account'] = $check_account;

        return $this;
    }

    public function getCheckNumber()
    {
        return $this->container['check_number'];
    }

    public function setCheckNumber($check_number)
    {
        $this->container['check_number'] = $check_number;

        return $this;
    }

    public function getCheckFirstName()
    {
        return $this->container['check_first_name'];
    }

    public function setCheckFirstName($check_first_name)
    {
        $this->container['check_first_name'] = $check_first_name;

        return $this;
    }

    public function getCheckLastName()
    {
        return $this->container['check_last_name'];
    }

    public function setCheckLastName($check_last_name)
    {
        $this->container['check_last_name'] = $check_last_name;

        return $this;
    }

    public function getNote()
    {
        return $this->container['note'];
    }

    public function setNote($note)
    {
        $this->container['note'] = $note;

        return $this;
    }

    public function getMaskedRouting()
    {
        return $this->container['masked_routing'];
    }

    public function setMaskedRouting($masked_routing)
    {
        $this->container['masked_routing'] = $masked_routing;

        return $this;
    }

    public function getMaskedAccount()
    {
        return $this->container['masked_account'];
    }

    public function setMaskedAccount($masked_account)
    {
        $this->container['masked_account'] = $masked_account;

        return $this;
    }

    public function getAccountType()
    {
        return $this->container['account_type'];
    }

    public function setAccountType($account_type)
    {
        $this->container['account_type'] = $account_type;

        return $this;
    }
    public function offsetExists($offset)
    {
        return isset($this->container[$offset]);
    }

    public function offsetGet($offset)
    {
        return isset($this->container[$offset]) ? $this->container[$offset] : null;
    }

    public function offsetSet($offset, $value)
    {
        if (is_null($offset)) {
            $this->container[] = $value;
        } else {
            $this->container[$offset] = $value;
        }
    }

    public function offsetUnset($offset)
    {
        unset($this->container[$offset]);
    }

    public function __toString()
    {
        if (defined('JSON_PRETTY_PRINT')) { // use JSON pretty print
            return json_encode(
                ObjectSerializer::sanitizeForSerialization($this),
                JSON_PRETTY_PRINT
            );
        }

        return json_encode(ObjectSerializer::sanitizeForSerialization($this));
    }
}

File: CodeSamples/python/swagger_client/models/vault_check.py
# coding: utf-8

class VaultCheck(object):
    swagger_types = {
        'response': 'Response',
        'check_id': 'int',
        'main': 'bool',
        'check_routing': 'str',
        'check_account': 'str',
        'check_number': 'str',
        'check_first_name': 'str',
        'check_last_name': 'str',
        'note': 'str',
        'masked_routing': 'str',
        'masked_account': 'str',
        'account_type': 'str'
    }

    attribute_map = {
        'response': 'Response',
        'check_id': 'CheckID',
        'main': 'Main',
        'check_routing': 'CheckRouting',
        'check_account': 'CheckAccount',
        'check_number': 'CheckNumber',
        'check_first_name': 'CheckFirstName',
        'check_last_name': 'CheckLastName',
        'note': 'Note',
        'masked_routing': 'MaskedRouting',
        'masked_account': 'MaskedAccount',
        'account_type': 'AccountType'
    }

    def __init__(self, response=None, check_id=None, main=None, check_routing=None, check_account=None, check_number=None, check_first_name=None, check_last_name=None, note=None, masked_routing=None, masked_account=None, account_type=None):  # noqa: E501
        """VaultCheck - a model defined in Swagger"""  # noqa: E501
        self._response = None
        self._check_id = None
        self._main = None
        self._check_routing = None
        self._check_account = None
        self._check_number = None
        self._check_first_name = None
        self._check_last_name = None
        self._note = None
        self._masked_routing = None
        self._masked_account = None
        self._account_type = None
        self.discriminator = None
        if response is not None:
            self.response = response
        if check_id is not None:
            self.check_id = check_id
        if main is not None:
            self.main = main
        if check_routing is not None:
            self.check_routing = check_routing
        if check_account is not None:
            self.check_account = check_account
        if check_number is not None:
            self.check_number = check_number
        if check_first_name is not None:
            self.check_first_name = check_first_name
        if check_last_name is not None:
            self.check_last_name = check_last_name
        if note is not None:
            self.note = note
        if masked_routing is not None:
            self.masked_routing = masked_routing
        if masked_account is not None:
            self.masked_account = masked_account
        if account_type is not None:
            self.account_type = account_type

    @property
    def response(self):
        return self._response

    @response.setter
    def response(self, response):
        self._response = response

    @property
    def check_id(self):
        return self._check_id

    @check_id.setter
    def check_id(self, check_id):
        self._check_id = check_id

    @property
    def main(self):
        return self._main

    @main.setter
    def main(self, main):
        self._main = main

    @property
    def check_routing(self):
        return self._check_routing

    @check_routing.setter
    def check_routing(self, check_routing):
        self._check_routing = check_routing

    @property
    def check_account(self):
        return self._check_account

    @check_account.setter
    def check_account(self, check_account):
        self._check_account = check_account

    @property
    def check_number(self):
        return self._check_number

    @check_number.setter
    def check_number(self, check_number):
        self._check_number = check_number

    @property
    def check_first_name(self):
        return self._check_first_name

    @check_first_name.setter
    def check_first_name(self, check_first_name):
        self._check_first_name = check_first_name

    @property
    def check_last_name(self):
        return self._check_last_name

    @check_last_name.setter
    def check_last_name(self, check_last_name):
        self._check_last_name = check_last_name

    @property
    def note(self):
        return self._note

    @note.setter
    def note(self, note):
        self._note = note

    @property
    def masked_routing(self):
        return self._masked_routing

    @masked_routing.setter
    def masked_routing(self, masked_routing):
        self._masked_routing = masked_routing

    @property
    def masked_account(self):
        return self._masked_account

    @masked_account.setter
    def masked_account(self, masked_account):
        self._masked_account = masked_account

    @property
    def account_type(self):
        return self._account_type

    @account_type.setter
    def account_type(self, account_type):
        self._account_type = account_type
